src/event-detail/ui/event-detail.html

```diff
@@ -127,7 +127,7 @@ <h1 class="title" ng-bind="::event.name"></h1>
                 <td><span class="row-label" translate-once="preview.where"></span></td>
                 <td><span translate-once="preview.online"></span></td>
               </tr>
-              <tr>
+              <tr ng-show="::event.attendanceMode != 'offline'">
```

Comment:
In js it is best practice to use **strict** equality operators.

```suggestion
              <tr ng-show="::event.attendanceMode !==  'offline'">
```

Also I think it makes more sense to write it like this.

```suggestion
              <tr ng-show="::event.attendanceMode === 'online'">
```

Reply:
The attendance mode can also be mixed, so `!== 'offline'` seems easier.

Reply:
Ah ok make sense in that case 👍